config/jobs/kubernetes/sig-node/dynamic-resource-allocation.conf

```diff
@@ -0,0 +1,65 @@
+[DEFAULT]
+interval = 6h
+testgrid_dashboards = sig-node-dynamic-resource-allocation
+testgrid_alert_email = eduard.bartosh@intel.com,patrick.ohly@intel.com
+timeout = 90m
+use-dind = false
+label_filter = Feature: containsAny DynamicResourceAllocation && Feature: isSubsetOf { Beta, DynamicResourceAllocation } && !Flaky
+optional = true
+
+# This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently beta)
+# on a kind cluster with containerd updated to a version with CDI support.
+[kind-dra]
+description = Runs E2E tests for Dynamic Resource Allocation beta features against a Kubernetes master cluster created with sigs.k8s.io/kind
+testgrid_alert_email = patrick.ohly@intel.com
+use_dind = true
+run_if_changed = /(dra|dynamicresources|resourceclaim|deviceclass|resourceslice|resourceclaimtemplate|dynamic-resource-allocation|pkg/apis/resource|api/resource)/.*.go
+
+# This jobs runs e2e.test with a focus on tests for the Dynamic Resource Allocation feature (currently alpha, soon beta)
+# on a kind cluster with containerd updated to a version with CDI support.
+#
+# Compared to ci-kind-dra, this one enables all DRA-related features.
+[kind-dra-all]
+description = Runs E2E tests for Dynamic Resource Allocation alpha and beta features against a Kubernetes master cluster created with sigs.k8s.io/kind
+testgrid_alert_email = patrick.ohly@intel.com
+use_dind = true
+run_if_changed = /(dra|dynamicresources|resourceclaim|deviceclass|resourceslice|resourceclaimtemplate|dynamic-resource-allocation|pkg/apis/resource|api/resource)/.*.go
+
+# This job runs e2e_node.test with a focus on tests for the Dynamic Resource Allocation feature (currently beta)
+[node-e2e-cgrpv1-crio-dra]
+job_type = node
+description = Runs E2E node tests for Dynamic Resource Allocation beta features with CRI-O using cgroup v1
+testgrid_dashboards = sig-node-cri-o, sig-node-dynamic-resource-allocation
+skip_report = false
+image_config_file = /home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/latest/image-config-cgroupv1-serial.yaml
+inject_ssh_public_key = true
```

Comment:
Same here: this can depend on the job type in the template.

Reply:
I don't think it can. not all presubmit jobs have this. It depends on a distro/image as far as I remember.